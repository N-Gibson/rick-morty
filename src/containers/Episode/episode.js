import React from 'react';

const Episode = ({ name, episode, air_date }) => {
  return (
    <div className='episode-card'>
      <h2>{name}</h2>
      <p>Episode: {episode}</p>
      <p>Air Date: {air_date}</p>
    </div>
  )
}

export default Episode