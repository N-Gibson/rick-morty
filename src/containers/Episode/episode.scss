.episode-card {
  background-image: url('https://media.giphy.com/media/hsITL773u90X6KprRR/giphy.gif');
  background-size: auto;
  border-radius: 5px;
  padding: 10px;
    &:hover {
      box-shadow: 0px 0px 10px 2px rgb(140, 244, 46);
    }
}