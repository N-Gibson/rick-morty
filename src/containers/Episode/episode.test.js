import React from 'react';
import Episode from '../Episode/episode';
import { shallow } from 'enzyme';

describe('Episode', () => {
  it('should match snapshot', () => {
    const wrapper = shallow(<Episode />)
    expect(wrapper).toMatchSnapshot()
  })
});