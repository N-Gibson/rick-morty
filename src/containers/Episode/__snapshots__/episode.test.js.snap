// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Episode should match snapshot 1`] = `ShallowWrapper {}`;
